lib/validation/rules/maxFileSize.ts

```diff
@@ -1,16 +1,15 @@
 import { helpers } from '@vuelidate/validators'
+import { createRule } from '../Rule'
 import { maxFileSize as baseMaxFileSize } from '../validators/maxFileSize'
 
+export const message = {
+  en: (size: string) => `The file must be smaller than or equal to ${size}.`,
+  ja: (size: string) => `ファイルは${size}よりも小さい必要があります。`
+}
+
 export function maxFileSize(size: string, msg?: string) {
-  return helpers.withParams(
-    { size },
-    helpers.withMessage(
-      ({ $params }) => {
-        return msg ?? `The file must be smaller than ${$params.size}.`
-      },
-      (value: File) => {
-        return !helpers.req(value) || baseMaxFileSize(value, size)
-      }
-    )
-  )
+  return createRule({
+    message: ({ lang }) => msg ?? message[lang](size),
+    validation: (value: File) => !helpers.req(value) || baseMaxFileSize(value, size)
```

Comment:
`!helpers.req(value)` may not be needed according to `createRule` implementation, but not so sure.

Reply:
Fixed: 5c20e70